elasticsearch/src/test/java/com/tcdi/zombodb/query_parser/TestQueryRewriter.java

```diff
@@ -148,6 +148,17 @@ public Map answer(InvocationOnMock invocationOnMock) throws Throwable {
                             fieldProps.put("analyzer", "fulltext");
 
                             fieldProps = new HashMap();
+                            props.put("sent", fieldProps);
+                            fieldProps.put("analyzer", "date");
+
+                            fieldProps = new HashMap();
+                            props.put("sent", fieldProps);
+                            fieldProps.put("analyzer", "date");
+                            final Map fieldsMap = new HashMap();
+                            fieldsMap.put("date", "something about date");
+                            fieldProps.put("fields", fieldsMap);
```

Comment:
I didn't see a unit test for tally() if the field is a date but this should open that up now.